### DISCUSSION TITLE
Integrating PostGIS Data with PRISM

### DISCUSSION THREAD

**Discussion by jeremi** [WFP-VAM/prism-frontend#1104] [General]
Hi @ericboucher,

Is there any plan or method to integrate PostGIS data with the WFP PRISM Frontend? Given its current setup, leveraging PostGIS could enable integration with [OpenSPP](https://github.com/OpenSPP) for enhanced visualization. 

Any guidance or thoughts on this would be appreciated.

Thanks.

**Comment by ericboucher** [comment]
@jeremi that's a neat idea. We're not really using discussions on this repo, would you mind opening an issue for this and descriving a bit more your usecase?
